.jshintrc

```diff
@@ -14,6 +14,7 @@
   "forin": false,
   "immed": false,
   "laxbreak": false,
+  "loopfunc": true,
```

Comment:
also noticed we don't have a "return" keyword here anymore - is that implicit now?